python/gps/algorithm/policy_opt/config.py

```diff
@@ -38,12 +38,9 @@
 POLICY_OPT_CAFFE.update(GENERIC_CONFIG)
 
 
-checkpoint_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
```

Comment:
self.checkpoint_file isn't called anywhere. A few commits ago it was used with self.save.save but now the weights are saved in manner that doesn't require a checkpoint_file. Do you plan on returning that functionality?

self.saver.save